spec/fixtures/proofing/lexis_nexis/ddp/request.json

```diff
@@ -17,7 +17,8 @@
   "policy": "test-policy",
```

Comment:
looks good.  not a blocker but we might want to update the sample response here which has ssn_hash_result : spec/fixtures/proofing/lexis_nexis/ddp/successful_response.json

Reply:
Ooo is that used by a spec?

Reply:
made some updates to it!